minifi-bootstrap/src/main/java/org/apache/nifi/minifi/bootstrap/RunMiNiFi.java

```diff
@@ -1197,7 +1197,14 @@ public void start() throws IOException, InterruptedException {
         this.changeListener = new MiNiFiConfigurationChangeListener(this, defaultLogger);
         this.periodicStatusReporters = initializePeriodicNotifiers();
         startPeriodicNotifiers();
-        this.changeCoordinator = initializeNotifier(this.changeListener);
+        try {
+            this.changeCoordinator = initializeNotifier(this.changeListener);
+        } catch (Exception e) {
```

Comment:
Those are fair points. My main point was that this way didn't offer any stacktrace in the event we hit an error we didn't expect. Instead of a more fine-grain catch, could we change the "cmdLogger" line to log the stack trace?